src/openqaoa-core/utilities.py

```diff
@@ -1,36 +1,20 @@
-#   Copyright 2021 Entropica Labs
-#
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
-#
-#       http://www.apache.org/licenses/LICENSE-2.0
-#
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-
 """
 Utility and convenience functions for a number of QAOA applications.
 """
 
 from __future__ import annotations
-
 from typing import Optional, Union, List, Tuple
 import itertools
 import numpy as np
 import uuid
 import matplotlib.pyplot as plt
 import networkx as nx
 
-from .qaoa_parameters import Hamiltonian, PauliOp, QAOAVariationalBaseParams
-from .qaoa_parameters.gatemap import TwoQubitRotationGateMap
+from .qaoa_components import Hamiltonian, PauliOp, QAOAVariationalBaseParams
+from .qaoa_components.ansatz_constructor.gatemap import TwoQubitRotationGateMap
```

Comment:
I wouldn't import anything from openqaoa here. If any function requires any openqaoa import then I would put that function in a `helper_function.py` file in the respective folder. If we import from openqaoa here then we can have problems of circular imports.

Reply:
Agreed. I'll look for how we can separate the functions that require said imports.

Reply:
Remove QAOAVariationalBaseParams and TwoQubitRotationGateMap.
Keep the remaining. To come back in the future.

Reply:
Resolved in commit [160d40c](https://github.com/entropicalabs/openqaoa/pull/170/commits/160d40cdd9d15920e71d78b81d34e1a8129b8ccd)